pkg/corerp/backend/deployment/deploymentprocessor_test.go

```diff
@@ -299,80 +299,6 @@ func Test_Render(t *testing.T) {
 		},
 	}
 
-	t.Run("verify render success", func(t *testing.T) {
-		mocks := setup(t)
-		dp := deploymentProcessor{mocks.model, mocks.databaseClient, nil, nil}
-
-		testResource := getTestResource()
-		testRendererOutput := getTestRendererOutput()
-		resourceID := getTestResourceID(testResource.ID)
-
-		depId1, _ := resources.ParseResource("/subscriptions/test-subscription/resourceGroups/test-resource-group/providers/Applications.Datastores/mongoDatabases/test-mongo")
```

Comment:
Mongo resource is a dependency here, so this should continue to be a valid scenario